app/exporters/manual_publishing_api_exporter.rb

```diff
@@ -1,11 +1,14 @@
 class ManualPublishingAPIExporter
+  UPDATE_TYPES = %w(minor major republish).freeze
```

Comment:
We're using the same set of update types here in the exporter and as well in the section publisher and api publisher. Might it be worthwhile having them defined in just one place? Or are we keeping them separate because we might be expecting variation between publishing/exporting services?